### PR TITLE
Replace deprecated BasicClientRequestResponseChannel with new NettyClientRequestResponseChannel

### PR DESCRIPTION
**Pull request by alexguzun**

### CODE REVIEW

File: src/test/java/io/vlingo/http/resource/ServerTest.java [hunk 1]
```diff
@@ -179,7 +178,7 @@ public void setUp() throws Exception {
 
     consumer = world.actorFor(ResponseChannelConsumer.class, Definition.has(TestResponseChannelConsumer.class, Definition.parameters(progress)));
 
-    client = new BasicClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240, world.defaultLogger());
+    client = new NettyClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240);
```
Comment (VaughnVernon) [inline src/test/java/io/vlingo/http/resource/ServerTest.java:181]: @alexguzun I didn't notice before, but this doesn't use the World logger. Is there a specific reason?

File: src/test/java/io/vlingo/http/resource/StaticFilesResourceTest.java [hunk 2]
```diff
@@ -182,7 +181,7 @@ public void setUp() throws Exception {
 
     progress = new Progress();
     consumer = world.actorFor(ResponseChannelConsumer.class, Definition.has(TestResponseChannelConsumer.class, Definition.parameters(progress)));
-    client = new BasicClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240, world.defaultLogger());
+    client = new NettyClientRequestResponseChannel(Address.from(Host.of("localhost"), serverPort, AddressType.NONE), consumer, 100, 10240);
```
Comment (VaughnVernon) [inline src/test/java/io/vlingo/http/resource/StaticFilesResourceTest.java:184]: World logger?

File: src/test/java/io/vlingo/http/resource/feed/FeedResourceTest.java [hunk 3]
```diff
@@ -91,7 +90,7 @@ public void setUp() throws Exception {
 
     progress = new Progress();
     consumer = world.actorFor(ResponseChannelConsumer.class, Definition.has(TestResponseChannelConsumer.class, Definition.parameters(progress)));
-    client = new BasicClientRequestResponseChannel(Address.from(Host.of("localhost"), testServerPort, AddressType.NONE), consumer, 100, 10240, world.defaultLogger());
+    client = new NettyClientRequestResponseChannel(Address.from(Host.of("localhost"), testServerPort, AddressType.NONE), consumer, 100, 10240);
```
Comment (VaughnVernon) [inline src/test/java/io/vlingo/http/resource/feed/FeedResourceTest.java:93]: World logger?

File: src/test/java/io/vlingo/http/resource/ServerTest.java (hunk 1 above)
Comment (alexguzun) [reply src/test/java/io/vlingo/http/resource/ServerTest.java:181]: For the Netty client implementation, I have decided, as previously discussed when talking about the issues of the current logging implementation, to use the SLF4J logger directly:

See  `io.vlingo.wire.fdx.bidirectional.netty.client.NettyClientRequestResponseChannel`
```
private final static Logger logger = LoggerFactory.getLogger(NettyClientRequestResponseChannel.class)
```

This allows configuring the the logging level specifically for the client:
` <logger name="io.vlingo.wire.fdx.bidirectional.netty.client" level="DEBUG"/>`

Also using the SLF4J logger directly captures the exact timestamp of the events, which is crucial when debugging issues.

Comment (VaughnVernon) [reply src/test/java/io/vlingo/http/resource/ServerTest.java:181]: Ok, sorry I didn't remember that. Approving.

**Pull request merged**